webplugin/knowledgebase/kb.js

```diff
@@ -47,6 +47,7 @@
                 KommunicateKB.getFaqs({
                     data: options.data,
                     success: function (response) {
+                        if(!response || !response.data) return null;
```

Comment:
add { } even if it is one line, that increases readability

Reply:
ok sir